pandas/tests/test_multilevel.py

```diff
@@ -1828,6 +1828,18 @@ def test_duplicate_groupby_issues(self):
         result = s.groupby(s.index).first()
         self.assertEquals(len(result), 3)
 
+    def test_duplicate_mi(self):
```

Comment:
@jreback longshot: do you recall if, deep down, this test is for sortlevel (not `sort_index(level)`) or for `df.loc`?

Reply:
was for loc iirc